### DISCUSSION TITLE
Adjust delay subtitles not doing anything?

### DISCUSSION THREAD

**Discussion by tack32** [smplayer-dev/smplayer#657] [General]
Under "subtitles" "set delay" I'm adjusting it, for example +900 or -900 but neither changes when I see the subtitles. In this case, the subtitles take too long to show up after the action they describe. Thanks

**Comment by smplayer-dev** [comment]
Are you sure the subtitles aren't hardcoded?

**Reply by tack32** [reply]
Thanks for your reply. Actually I don't know what that means. I just saw it as an option under subtitles and thought it would be good   because the subtitles are behind the action in many things I watch. In this case it's a football game https://www.youtube.com/watch?v=4yjRZF5WuPo

**Comment by smplayer-dev** [comment]
Ok, I see, I'll investigate the problem.

**Comment by tack32** [comment]
That will be interesting to see if it's possible. Thanks!

**Comment by smplayer-dev** [comment]
I think the option does work. You need to set it to -7500 or so.

**Comment by tack32** [comment]
That's cool! I tried it. Thanks!
